godotproject/Level0.tscn

```diff
@@ -23,14 +23,11 @@ collision_layer = 34
 collision_mask = 17
 
 [node name="Camera2D" type="Camera2D" parent="Player"]
-position = Vector2( 0, 7.20007 )
-scale = Vector2( 1.05247, 1.26072 )
 current = true
 limit_left = 0
 limit_top = 0
 limit_right = 2000
 limit_bottom = 1000
-drag_margin_h_enabled = true
 drag_margin_v_enabled = true
```

Comment:
I removed the position and scale values, I'm not sure where they came from but we don't need them.

I turned off the horizontal drag margin so that the camera would better track the frog's movement.